sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Pickup Based Voxel Grid Downsample Filter Node",
+  "type": "object",
+  "definitions": {
+    "pickup_based_voxel_grid_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "voxel size along x-axis [m]",
+          "default": "1.0"
+        },
+        "voxel_size_y": {
```

Comment:
Can you add limitations to these values?
(applies to all other related PRs)

Reply:
Thanks, fixed them in https://github.com/autowarefoundation/autoware.universe/pull/8481/commits/40feff625de91cc41ea5406aacdce703ce6223fe. 

* I also update boundary in other PRs